auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/utils/JwtUtil.java

```diff
@@ -70,4 +71,12 @@ public static boolean isAuthorizationHeaderValid(String tokenPrefix, String head
         tokens.put(REFRESH_TOKEN, refreshToken);
         return tokens;
     }
+
+    public static Map<String, Object> getLoginResponseBody(User user, String accessToken, String refreshToken) {
+        Map<String, Object> response = new HashMap<>();
+        response.put(ACCESS_TOKEN, accessToken);
+        response.put(REFRESH_TOKEN, refreshToken);
+        response.put("credentials", user);
```

Comment:
Why not `user` instead of `credentials `?

Reply:
On frontend I used the key "user" to store the whole response object including the tokens and the user object, so I figured it'd be nicer to use credentials, then I can access the object as `storage.user.credentials`. I can refactor if it would make more sense to use `user`.